### ISSUE TITLE
Some rules don't work if @flow annotation is in a multiline comment

### ISSUE THREAD

**Issue by reosarevok** [flow-typed/eslint-plugin-ft-flow#56]
[no-flow-suppressions-in-strict-files](https://github.com/flow-typed/eslint-plugin-ft-flow?tab=readme-ov-file#no-flow-suppressions-in-strict-files) and [newline-after-flow-annotation](https://github.com/flow-typed/eslint-plugin-ft-flow?tab=readme-ov-file#newline-after-flow-annotation) at the very least do not seem to notice the `@flow strict` annotation if it's inside a multiline comment, such as:

```
/*
 * @flow strict
 * Copyright (C) 2017 MetaBrainz Foundation
 *
 * This file is part of MusicBrainz, the open internet music database,
 * and is licensed under the GPL version 2, or (at your option) any
 * later version: http://www.gnu.org/licenses/gpl-2.0.txt
 */
```

Having `{/* $FlowIgnore[sketchy-null-number] */}` in that same file does not trigger the rule, but it should.

**Comment by Brianzchen** [comment]
Hi @reosarevok thanks for raising the issues and sorry for getting back to you so late, I haven't been on github in a while. Your resolved issues can be found on version 3.0.9.

Let me know if you come across anymore issues!

**Comment by reosarevok** [comment]
Thanks! :)
